### DISCUSSION TITLE
"data": "writer buffer bellow minimum. Buffer size must be at least 1."

### DISCUSSION THREAD

**Discussion by Miguelklappes** [lnx-search/lnx#99] [Q&A] [answered]
Keep getting this error and the writer buffer is set to 3000000

**Comment by ChillFish8** [comment, marked as answer]
This looks to be a typo, in the error. The error is actually produced by the number of writer threads being specified being set to `0`.

If you haven't explicitly set the number of writer threads to be `0` then this is probably the result of LNX failing to get the number of available cores on your machine, which causes it to default to `0`. 

A simple fix to try is to add the `"writer_threads": 8` field in your schema creation payload, feel free to replace `8` with a number that's more suited to your machine (normally the number of logical CPU cores on your machine)

**Reply by Miguelklappes** [reply]
Yes, that was it, thank you!

**Reply by ChillFish8** [reply]
I'll make an issue to correct that typo 👍

**Comment by jazoom** [comment]
I got this same problem in a deploy on fly.io
